packages/editor-ui/src/components/TextEdit.vue

```diff
@@ -60,7 +60,7 @@ const closeDialog = () => {
 	<div v-if="dialogVisible">
 		<el-dialog
 			:model-value="dialogVisible"
-			append-to-body
+			append-to="#app-modals"
```

Comment:
We seem to be using this ID in several places, perhaps some constant would make sense to have a single source of truth